gateway/src/services/error-handler.ts

```diff
@@ -86,6 +86,7 @@ export const SWAP_PRICE_EXCEEDS_LIMIT_PRICE_ERROR_CODE = 1008;
 export const SWAP_PRICE_LOWER_THAN_LIMIT_PRICE_ERROR_CODE = 1009;
 export const SERVICE_UNITIALIZED_ERROR_CODE = 1010;
 export const UNKNOWN_CHAIN_ERROR_CODE = 1011;
+export const INVALID_NONCE_ERROR_CODE = 1012;
```

Comment:
We should update documentation for this.

Reply:
Which documentation are you talking about? If you're talking about Swagger, I dont see any error definitions anywhere.

Reply:
https://www.notion.so/hummingbot/API-Specs-ee319e13355a4906af6ab30bfe1496ea

Reply:
Included the new Error Codes in the Notion doc